hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -73,18 +79,42 @@ public static void main(String[] args) throws Exception {
         break;
       case IMPORT:
       case UPSERT:
-        assert (args.length == 11);
+        assert (args.length >= 12);
```

Comment:
I think you can assume that the props file is passed, no need for these validations, since we are not doing these validations for the other args anyways

Reply:
Let us discuss this on https://github.com/apache/incubator-hudi/pull/1080#discussion_r355368831. :)